syncany-cli/src/main/resources/org/syncany/cli/cmd/help.status.skel

```diff
@@ -2,28 +2,33 @@ NAME
   sy-status - list new and changed files in local Syncany folder
     
 SYNOPSIS
-  sy status [-f | --force-checksum] 
+  sy status [-f | --force-checksum]
   
-DESCRIPTION 
+DESCRIPTION
   This command compares the local file tree on the disk with the local
   database and detects local changes. These changes are printed to the
   console.
   
   Local changes are detected using the last modified date and the file size
   of a file. If they match the local database, the command assumes that the
-  content has not changed (no checksum comparison). If -f is enabled, the 
+  content has not changed (no checksum comparison). If -f is enabled, the
   checksum is additionally compared.
   
-  This command is used by the 'up' command to detect local changes.  
+  This command is used by the 'up' command to detect local changes.
   
 OPTIONS
   -f, --force-checksum
     Enforces this command to compare files by checksum, not by file size
     and last modified date only. This option is particularly useful if
-    files are modified in-place very often (last modified date and size 
+    files are modified in-place very often (last modified date and size
     do not change). For large local folders, this option can tremendously
-    decrease the performance of this command and increase I/O significantly.  
- 
+    decrease the performance of this command and increase I/O significantly.
+
+  -D, --no-delete
+    With tihs option, locally deleted files will be ignored. This option
```

Comment:
Also, the text is written as if this was an option for the `UpOperation` (which, obviously, it also is). But it should be written more like "This command doesn't *list* deleted files; and if used in 'up', it won't upload them."